### PR TITLE
tiny themefix, tutorialfix

### PR DESCRIPTION
**Pull request by brenthuisman**

### CODE REVIEW

File: doc/tutorial/mpi.rst [hunk 1]
```diff
@@ -64,8 +64,8 @@ cells over 5 nodes, simulating 100 cells each.
 
 .. code-block::
 
-   srun -n NRANKS python network_ring_mpi.py
-   mpirun -n NRANKS python network_ring_mpi.py
+   srun -n NRANKS python mpi.py
```
Comment (bcumming) [inline doc/tutorial/mpi.rst:67]: Should this be `${NRANKS}`, to make it bashy, and cut and paste freindly?

**Review by bcumming — CHANGES_REQUESTED** [review]

File: doc/tutorial/mpi.rst (hunk 1 above)
Comment (thorstenhater) [inline doc/tutorial/mpi.rst:67]: _also_ ``srun`` is specific to SLURM, not MPI, which might cause quite some confusion on a laptop w/o a resource manager.

**Review by thorstenhater — CHANGES_REQUESTED** [review]

File: doc/tutorial/mpi.rst (hunk 1 above)
Comment (brenthuisman) [reply doc/tutorial/mpi.rst:67]: I'm not expecting that variable is set, so I'd say it's good that the line will fail if left unedited. The context is quite clear about this, I think.

Comment (brenthuisman) [reply doc/tutorial/mpi.rst]: Aight.

Comment (bcumming) [reply doc/tutorial/mpi.rst]: Source code examples should be valid code. You could just say `mpirun -n 5 python mpi.py`.

Comment (brenthuisman) [reply doc/tutorial/mpi.rst]: done

**Review by thorstenhater — APPROVED** [review]

**Pull request merged**